public class exceptionsprac {

/*	public static void main(String[] args) {
		// TODO Auto-generated method stub
		int quotient;
		int numerator = 23;
		int denominator = 0;
		try{
		  quotient = numerator/denominator;
		  System.out.println("The answer is: " + quotient);
		} catch (ArithmeticException e){
		  System.out.println("Error: Division by zero");
		}
	} */
}